src/Misc/PresetExtractor.cpp

```diff
@@ -53,47 +53,6 @@ const rtosc::Ports real_preset_ports =
                         pre[i].type.c_str());
 
         }},
-    {"copy:s:ss:si:ssi", 0, 0,
-        [](const char *msg, rtosc::RtData &d) {
-            MiddleWare &mw = *(MiddleWare*)d.obj;
-            assert(d.obj);
-            std::string args = rtosc_argument_string(msg);
-            d.reply(d.loc, "s", "clipboard copy...");
-            printf("\nClipboard Copy...\n");
-            if(args == "s")
-                presetCopy(mw, rtosc_argument(msg, 0).s, "");
-            else if(args == "ss")
-                presetCopy(mw, rtosc_argument(msg, 0).s,
-                            rtosc_argument(msg, 1).s);
-            else if(args == "si")
-                presetCopyArray(mw, rtosc_argument(msg, 0).s,
-                            rtosc_argument(msg, 1).i, "");
-            else if(args == "ssi")
-                presetCopyArray(mw, rtosc_argument(msg, 0).s,
-                            rtosc_argument(msg, 2).i, rtosc_argument(msg, 1).s);
-            else
-                assert(false && "bad arguments");
-        }},
-    {"paste:s:ss:si:ssi", 0, 0,
-        [](const char *msg, rtosc::RtData &d) {
-            MiddleWare &mw = *(MiddleWare*)d.obj;
-            assert(d.obj);
-            std::string args = rtosc_argument_string(msg);
-            d.reply(d.loc, "s", "clipboard paste...");
-            if(args == "s")
-                presetPaste(mw, rtosc_argument(msg, 0).s, "");
-            else if(args == "ss")
-                presetPaste(mw, rtosc_argument(msg, 0).s,
-                            rtosc_argument(msg, 1).s);
-            else if(args == "si")
-                presetPasteArray(mw, rtosc_argument(msg, 0).s,
-                            rtosc_argument(msg, 1).i, "");
-            else if(args == "ssi")
-                presetPasteArray(mw, rtosc_argument(msg, 0).s,
-                            rtosc_argument(msg, 2).i, rtosc_argument(msg, 1).s);
-            else
-                assert(false && "bad arguments");
-        }},
     {"clipboard-type:", 0, 0,
```

Comment:
I'm going to have to find where those real copy/paste definitions migrated to, but it feels like there's an extra presetsObj which is being passed around to places where it only sometimes is going to be applicable? i.e. reading this file in isolation it could make sense for add synth and oscilgen, but not for other preset classes?

I think this is in the documentation, so I'll circle back to this with a better understanding, though this feels like a spot where I might add in a few tests to sanity check my understanding.